paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/SavedPaymentMethodTab.kt

```diff
@@ -174,6 +176,7 @@ private fun SavedPaymentMethodBadge(
 private fun SavedPaymentMethodCard(
     isSelected: Boolean,
     isEnabled: Boolean,
```

Comment:
it looks like this `isEnabled` is now only used as the default value for `isClickable`. Can we rename `isEnabled` to `isClickable` instead of having both?

Reply:
Ohh, good find! Thanks!